docs/_scripts/meetup_events.py

```diff
@@ -0,0 +1,64 @@
+import yaml
+import glob
+import io
+import meetup.api
+
+from time import strftime, gmtime
+
+from pprint import pprint
+
+def load_yaml(path):
+    with io.open(path, encoding='utf-8') as filepath:
+        return yaml.safe_load(filepath)
+
+def load_meetups():
+    result = []
+    for yaml_file in glob.glob('../_data/meetups/*.yaml'):
+        meetup = load_yaml(yaml_file)
+        if not 'website' in meetup:
+            raise SphinxError('Meetup needs a website')
+        result.append(meetup['website'])
+    return result
+
+
+meetups = load_meetups()
+
+client = meetup.api.Client()
+
+for meetup in meetups:
+    # Substring the meetup.com and trailing slash. EWWW
+    #print(m[23:-1])
+
+    t = client.GetEvents(group_urlname=meetup[23:-1])
+
+    for event in t.results:
+        # pprint('##'+event['group']['name'])
+        # pprint(event['name'])
+        # pprint(event['event_url'])
+
+        ##################################################
+        ms_since_the_epoch = event['time']
```

Comment:
also you can probably just make one call to strftime like

date = strftime("%B + %d", gmtime(event['time']))

Reply:
I've not checked that at all

Reply:
`date = strftime("%B %d", gmtime(event['time']))` works! 👍